### DISCUSSION TITLE
Welcome to awesome-android-kotlin-apps Discussions!

### DISCUSSION THREAD

**Discussion by androiddevnotes** [androiddevnotes/awesome-android-kotlin-apps#60] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by theapache64** [comment]
:muscle: :muscle: :muscle: :muscle: :muscle: :muscle:

**Comment by GithinjiHans** [comment]
Am a C++ and Kotlin dev...and tbh sometimes handling projects alone is boring. so we can build projects as a community and learn from each other I guess.
